submission/example_method.py

```diff
@@ -1,10 +1,15 @@
 """ Demo: Creates a simple new method and applies it to a single CL setting.
+
+To run this example on one the Settings defined below:
+```console
+$ python submission/example_method.py
+```
```

Comment:
Wait, what? No! Run `make sl`.  That's confusing people.

Reply:
These aren't instructions on how to run or debug the track, its for quickly debugging the Method, on the settings defined at the bottom. Take a look at the end of the file, I think it'll make more sense. There might be a better way to phrase this though.

Reply:
Yes and I'm saying you're confusing people with this. These kinds of examples can live in Sequoia. This submission seed repo has one main interface and that's the makefile.